gramps/gui/plug/_windows.py

```diff
@@ -1217,7 +1217,7 @@ def install_addons(self, obj):
                      "%s %s" % (ngettext("{number_of} addon was installed.",
                                          "{number_of} addons were installed.",
                                          count).format(number_of=count),
-                                _("You need to restart Gramps to see new views.")),
+                        _("You need to restart Gramps to use the new addons.")),
```

Comment:
Note this was part of https://github.com/gramps-project/gramps/pull/63 and I split it out here.

Also I was just about to split out the  "Change config keys to mention addon" part leaving only the split addons part should I continue?